views/tx.tmpl

```diff
@@ -59,6 +59,16 @@
                         {{if ne .TicketInfo.SpendStatus "voted"}} {{.TicketInfo.PoolStatus}}  / {{end}} {{if (index .SpendingTxns 0).Hash}}<a href="/tx/{{(index .SpendingTxns 0).Hash}}">{{.TicketInfo.SpendStatus}}</a>{{else}}{{.TicketInfo.SpendStatus}}{{end}}
                     </td>
                 </tr>
+                {{if .TicketInfo.TicketExpiryBlocksLeft}}
+                <tr>
+                    <td class="text-right pr-2 h1rem p03rem0">TICKET EXPIRY</td>
+                    <td>{{.TicketInfo.TicketExpiryBlocksLeft}} blocks or (~{{printf "%.2f" .TicketInfo.TicketExpiryDaysLeft}} Days Left)</td>
+                </tr>
+                <tr>
+                    <td class="text-right pr-2 h1rem p03rem0">CLOSURE TO AVERAGE WIN TIME</td>
```

Comment:
I think the word closure here is confusing.  Does this mean remaining time?

Reply:
I use closure here to depict how close the ticket is to the average win time, the remain time till it gets there

Reply:
Couldn't really come up with a better word there